<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>bwCSV</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QTableWidget" name="tblContents">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>true</bool>
      </attribute>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>15</number>
      </property>
      <item>
       <widget class="QLabel" name="lblSearch">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Search:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtSearch">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkCaseInsensitive">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Case insensitive</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkWholeWord">
        <property name="text">
         <string>Whole word search</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="vspcrSearch1">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>200</width>
          <height>15</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lblColumns">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Search columns:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="lstColumns">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="vspcrSearch2">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>200</width>
          <height>15</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lblResults">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Search results:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="lstSearchResults">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="menuOpen"/>
    <addaction name="menuQuit"/>
   </widget>
   <widget class="QMenu" name="menuSettings_2">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="menuFieldSeparator"/>
    <addaction name="menuTextDelimiter"/>
    <addaction name="separator"/>
    <addaction name="menuHorizontalHeader"/>
    <addaction name="menuVerticalHeader"/>
    <addaction name="separator"/>
    <addaction name="menuHeaderLabels"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="menuAboutMessage"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings_2"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_file">
   <property name="text">
    <string>Open file...</string>
   </property>
  </action>
  <action name="actionClose_file">
   <property name="text">
    <string>Close file</string>
   </property>
  </action>
  <action name="menuQuit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionComma">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Comma</string>
   </property>
  </action>
  <action name="actionSemicolon">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Semicolon</string>
   </property>
  </action>
  <action name="actionSpace">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Space</string>
   </property>
  </action>
  <action name="actionTab">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tab</string>
   </property>
  </action>
  <action name="actionDouble_quote">
   <property name="text">
    <string>Double-quote</string>
   </property>
  </action>
  <action name="actionSingle_quote">
   <property name="text">
    <string>Single-quote</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences...</string>
   </property>
  </action>
  <action name="menuSettings">
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
  <action name="menuOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="menuFieldSeparator">
   <property name="text">
    <string>Field separator...</string>
   </property>
  </action>
  <action name="menuTextDelimiter">
   <property name="text">
    <string>Text delimiter...</string>
   </property>
  </action>
  <action name="menuVerticalHeader">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Vertical header</string>
   </property>
  </action>
  <action name="menuHorizontalHeader">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Horizontal header</string>
   </property>
  </action>
  <action name="menuHeaderLabels">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Set header labels from first line</string>
   </property>
  </action>
  <action name="menuAboutMessage">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tblContents</tabstop>
  <tabstop>txtSearch</tabstop>
  <tabstop>chkCaseInsensitive</tabstop>
  <tabstop>lstColumns</tabstop>
  <tabstop>lstSearchResults</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
